password_security/views/res_config_settings_views.xml

```diff
@@ -14,10 +14,10 @@
         <field name="arch" type="xml">
             <!-- We hide Odoo's minlength sections so it does not create confusion -->
             <xpath expr="//label[@for='minlength']/../.." position="attributes">
-                <attribute name="style">display: none</attribute>
+                <attribute name="invisible">1</attribute>
```

Comment:
Seems weird to continue to hide `minlength` now that it has been adopted over `password_length`.

Reply:
It's just because I wanted to embed the `minlength` inside the group of fields "Password Policy"

https://github.com/OCA/server-auth/blob/6eab9a5d01c5fa49263ea9aba33bade290bfc9cf/password_security/views/res_config_settings_views.xml#L53-L57

This way it looks&feels better, otherwise the `minlength` will stay detached from the rest of the fields.

Reply:
How about instead of removing the entire box that contains the label, we reuse it? There might have been some customization within the settings box that gets removed otherwise.

Something like this?
```
<!-- Remove label for uniform styling -->
<xpath expr="//label[@for='minlength']" position="replace"/>


<!-- Add an id for easier lookup -->
<xpath expr="//field[@name='minlength']/ancestor::div[hasclass('o_setting_box')]" position="attributes">
    <attribute name="id">password_policy</attribute>
</xpath>

<!-- Move the settings box to desired location -->
<xpath expr="//div[@id='enable_password_reset']" position="after">
    <xpath expr="//div[@id='password_policy']" position="move"/>
</xpath>


<!-- Ensure our settings will come before any former customization -->
<xpath expr="//div[@id='password_policy']//div[hasclass('o_setting_right_pane')]/*" position="before">
    <!-- Add all the settings here -->
    <span id="minlength">
         Minimum number of characters
    </span>
</xpath>


<!-- Add oe_inline attribute to minlength -->
<xpath expr="//field[@name='minlength']" position="attributes">
  <attribute name="class">oe_inline</attribute>
</xpath>

<!-- Move the minlength field to desired location -->
<xpath expr="//span[@id='minlength']" position="inside">
    <xpath expr="//field[@name='minlength']" position="move"/>
</xpath>
```

It might seem like a lot of work, but I personally feel like we should try to minimize making nodes disappear

Reply:
Looks mostly good to me! There's the comment that still says <!-- Ensure our settings will come _before_ any former customization -->

I don't know if you wanted the password_security settings to come before or after the previous nodes.